app/views/layouts/_header.html.erb

```diff
@@ -0,0 +1,12 @@
+<header class="navbar navbar-fixed-top navbar-inverse">
+  <div class="container">
+    <%= link_to "sample app", '#', id: "logo" %>
+    <nav>
+      <ul class="nav navbar-nav navbar-right">
+        <li><%= link_to "Home",   home_path %></li>
+        <li><%= link_to "Help",   help_path %></li>
+	<li><%= link_to "Log in", '#' %></li>
```

Comment:
fix indent

Reply:
vim didn't show a difference but turns out there was a tab mixed in with the spaces.